public/clinics.js

```diff
@@ -85,6 +100,35 @@ function createCompanyAddressSection(index, id, name, address) {
         addressInput.value = address;
     }
     section.appendChild(addressInput);
+    section.appendChild(document.createElement('br'));
+
+    const companyPostCodeLabel = document.createElement('label');
+    companyPostCodeLabel.textContent = 'Company Postcode:';
+    section.appendChild(companyPostCodeLabel);
+    section.appendChild(document.createElement('br'));
+    const companyPostCode = document.createElement('input');
+    companyPostCode.type = 'text';
+    companyPostCode.classList.add('companyPostCode');
+    companyPostCode.name = 'companyPostCode';
+    if (postcode) {
+        companyPostCode.value = postcode;
+    }
+    section.appendChild(companyPostCode);
+    section.appendChild(document.createElement('br'));
+
+    const lineNumberLabel = document.createElement('label');
+    lineNumberLabel.textContent = 'Line Number:';
+    section.appendChild(lineNumberLabel);
+    section.appendChild(document.createElement('br'));
+    const lineNumber = document.createElement('input');
+    lineNumber.type = 'text';
+    lineNumber.classList.add('lineNumber');
```

Comment:
None of the names used comply with the specification. Line number for example should be called "Accounting Line for Service Code"